src/journeys/organisationTypeService.js

```diff
@@ -0,0 +1,31 @@
+function validateOrganisationType(req, res) {
+  try {
+    const { organisationType } = req.body;
+    const queryParams = req.query;
+    const editMode = queryParams.editMode;
+
+    if (
```

Comment:
We can't use these conditions. They are using hardcoded values list.
Plus, if a dev modify the value in the form, he/she will never think about to change in the validation function.
Maybe it's better to think in the opposite way, which condition(s) will trigger an error?

Reply:
Yeah I knoww but  thought it would be a bit much for our app as  in our case, all the checks would lead to the same action – moving to the next page ? So I should add if checks for every single value ?

Reply:
No, you don't have to check what is the selected value. For example, you're not doing it for other validation functions.
The goal of your "if" is to check if you have a value (whatever the value). If not: error.

Reply:
ohhh wait I think I misunderstood your comment, you are saying I should not have done if check with comparing our values , ok will change that .